app/konnect/org-management/okta-idp.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Set Up Okta IdP Authentication
+no_version: true
+badge: enterprise
+---
+
+As an alternative to Konnect Cloud’s native basic authentication, you can set up
+org access through Okta. This way, your users can log in to Konnect Cloud
+using their Okta credentials, and without needing a separate login.
+
+You can't mix authenticators in Konnect Cloud. With Okta IdP
+authentication enabled, all non-admin Konnect users will subsequently log in
+through Okta. Only Konnect admins can continue to log in with basic
+authentication.
+
+> This feature is only available with Kong Konnect Enterprise.
+
+## Prerequisites
+* [**Organization Admin**](/konnect/reference/org-management/#role-definitions)
+permissions in Konnect Cloud
+* [Okta developer account](https://developer.okta.com/)
+
+## Set up Okta IdP Auth
+
+### Prepare the Okta application
+
+Create a new application in Okta to manage Konnect Cloud account integration.
+
+1. Sign in to your [Okta developer account](https://developer.okta.com/).
+1. From the left menu, select **Applications**, then **Create Integration**.
+1. Select the application type:
+    1. Under **Sign-on method**, select **OIDC - OpenID Connect**.
+    1. Under **Application Type**, select **Web Application**.
+1. Click **Next**. Configure the application:
+    1. Create a unique name for your application.
+    1. Under **Grant Type**, select **Authorization Code**.
+    1. In both the **Sign-in redirect URIs** and
+**Sign-out redirect URIs** fields, enter:
+
+      `https://<div contenteditable="true">{YOUR_KONNECT_APP_DOMAIN}</div>/login`.
+
+      Replace `{YOUR_KONNECT_APP_DOMAIN}` with your own domain.
+1. Save your settings to generate connection details.
+
+    Leave this page open. You'll need the details here to configure your Konnect
+    Cloud account.
+
+### Set the default IdP for Konnect Cloud
+
+1. In a separate browser tab, log in to [Konnect Cloud](https://konnect.konghq.com).
+1. Open ![settings icon](/assets/images/icons/konnect/konnect-settings.svg){:.inline .no-image-expand}
+**Settings**, then **Identity Management**.
+1. Select **Okta**.
+1. Copy and paste the **Client ID** and **Client Secret** from your Okta
+application into Konnect Cloud.
+
+    See the [Okta developer documentation](https://developer.okta.com/docs/guides/find-your-app-credentials/findcreds/)
+    to learn more about client credentials in Okta.
+
+1. Create the **Issuer URL**:
+    1. Copy the **Okta domain** from your Okta application.
+    1. Paste it into the **Issuer** field in Konnect using the following format:
+
+        `https://<div contenteditable="true">{OKTA_DOMAIN}</div>/oauth2/default`
+1. Save your changes.
+1. Copy the generated login URL.
```

Comment:
This URL will no longer be generated for users. We will allow admins to set a specific login URL that they define. This is a new feature that is being added (not yet in dev, though). Nate and Dustin are working on this now.

Reply:
Will update this step once I get concrete info on this one.

Reply:
What are the format requirements for this field? Is this path going to be appended to their domain, or do they need to include their org domain in the path? 
For example, can I enter `/login` or do I need to put `myorg-konnect.konghq.com/login`?

Reply:
From ticket: How do I know what my organizational login path is? Answer: this can be any value you like, for example, vanguard-dev results in https://konnect.konghq.com/login/vanguard-dev. The only restriction is that it be unique (no other orgs using it) and non-null (field is required)